src/components/FilterPanel/FilterPanel.js

```diff
@@ -1,107 +1,82 @@
 /**
- * @file Filter Panel.
- * @copyright IBM Security 2019
+ * @file Filter panel component.
+ * @copyright IBM Security 2020
  */
 
 import React from 'react';
 import PropTypes from 'prop-types';
-import FilterSearch from './FilterSearch';
-import FilterCategory from './FilterCategory';
-import {
-  filterDataPropTypes,
-  filterDataDefaultProps,
-  getFilterCategoriesArray,
-} from './FilterPanelUtilities';
+import classnames from 'classnames';
+
+import LegacyFilterPanel from './LEGACY_FilterPanel';
 import { getComponentNamespace } from '../../globals/namespace';
-import * as defaultLabels from '../../globals/nls';
+import deprecatedProp from '../../globals/prop-types';
+
+import FilterAccordion from './FilterAccordion';
+import FilterAccordionItem from './FilterAccordionItem';
+import FilterCheckbox from './FilterCheckbox';
+import FilterGroup from './FilterGroup';
+import FilterLabel from './FilterLabel';
+import FilterSearch from './FilterSearch';
```

Comment:
I do not think that you would want to import all of these components inside `FilterPanel` as some of them may not be used at all. If the consumer uses filterData and the `LegacyFilterPanel`, then they would not use any of the other subcomponents.

Reply:
That's a good point. I was trying to follow how the `DataTable` components are exported, which adds all subcomponents to the `DataTable` component object. @SimonFinney and @jendowns , do you have a preference?

Reply:
@diego-codes I'm cool with keeping this similar to the `DataTable` but the one thing I would ask is to keep `FilterPanel` as a default export, for backwards compatibility in case someone is currently importing it directly.

Reply:
Okay, I exported the `FilterPanel` as the default component. 

@jendowns it doesn't seem like you feel strongly for adding the subcomponents to the `FilterPanel` object. As I am looking through this, I agree with Alex. One of the benefits of creating composable components is that consumers only import what they need and ignore what they don't need. Adding subcomponents to the `FilterPanel` component would negate this benefit by shipping code to users' browsers that does not get used.

Reply:
@diego-codes Yeah that's right, I don't feel strongly about it & you both make a compelling argument. 👍